lib/peripheral.dart

```diff
@@ -66,6 +110,10 @@ class Peripheral {
         transactionId ?? TransactionIdGenerator.getNextId(),
       );
 
+  /// Write Characteristic value.
```

Comment:
```suggestion
  /// Writes the value of a [Characteristic] identified [characteristicUUID].
```

According to [third-person rule](https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs).